docs/component/rl.rst

```diff
@@ -0,0 +1,242 @@
+.. _rl:
+========================================================================
+Reinforcement Learning in Quantitative Trading
+========================================================================
+.. currentmodule:: qlib
+
+Reinforcement Learning
+============
+
+Reinforcement learning attempts to optimize an accumulative numerical reward signal by directly interacting with the environment under a few assumptions such as Markov Decision Process(MDP), which is the third basic machine learning method besides supervised and unsupervised learning.
+
+As demonstrated in the following figure, an RL system consists of four elements, 1)the agent 2) the environment the agent interacts with 3) the policy that the agent follows to take actions on the environment and 4)the reward signal from the environment to the agent. 
+In general, the agent can perceive and interpret its environment, take actions and learn through reward, to seek long-term and maximum overall reward to achieve an optimal solution.
+
+.. image:: ../_static/img/RL_framework.png
```

Comment:
I think the image might be too small. Have you checked it in the rendered document?